antora-ui-camel/src/css/doc.css

```diff
@@ -644,6 +644,10 @@ kbd,
   width: 100%;
 }
 
+.git-img {
+  width: 50px;
+  height: 50px;
+}
```

Comment:
Use this:

https://github.com/apache/camel-website/blob/60537f0c4d18dce038d2922a72644cc9342cf6ef/antora-ui-camel/src/css/blog.css#L128-L130

Put this change in `community.css` and use `.community img.avatar` as rule.